rubicon/ui/dashboard.py

```diff
@@ -23,12 +23,15 @@ class Dashboard:
     page_size : int, optional
         The number of rows that will be displayed on a page within the
         experiment table.
+    storage_options : dict, optional
+        Additional keyword arguments specific to the protocol being chosen. They
+        are passed directly to the underlying filesystem class.
     """
 
-    def __init__(self, persistence, root_dir=None, page_size=10):
+    def __init__(self, persistence, root_dir=None, page_size=10, **storage_options):
```

Comment:
woulda been a few less files to implement this in if we were passing the `Rubicon` object thru like Mike suggested yesterday lol